diff --git a/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/ReanimatedMessageQueueThreadBase.java b/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/ReanimatedMessageQueueThreadBase.java
index 2216d33..ce973e1 100644
--- a/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/ReanimatedMessageQueueThreadBase.java
+++ b/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/ReanimatedMessageQueueThreadBase.java
@@ -55,4 +55,9 @@ public abstract class ReanimatedMessageQueueThreadBase implements MessageQueueTh
   @Override
   public void resetPerfStats() {
     messageQueueThread.resetPerfStats();
   }
+
+  @Override
+  public boolean isIdle() {
+    return messageQueueThread.isIdle();
+  }
 }
